src/shared/actions/events/event.js

```diff
@@ -32,6 +32,12 @@ export function fetchFailure(error) {
 
 export const fetchEvent = (fetch) => (
   (dispatch, getState, nextState) => {
+    const storedEvent = getState().event;
+
+    if (storedEvent && Object.keys(storedEvent).length > 0) {
```

Comment:
It checks for events though, rather than here where we're looking for a single event